src/runtime/node/internal/process/process.ts

```diff
@@ -5,6 +5,8 @@ import {
   createNotImplementedError,
 } from "../../../_internal/utils.ts";
 
+const NODE_VERSION = "22.14.0";
```

Comment:
Might be worth adding the comment about this needing to be a reasonably new version in the code for future reference.

Reply:
we have reference version we sync unenv with in nvmrc